packages/stream-metadata/tests/testUtils.ts

```diff
@@ -102,13 +104,21 @@ export async function makeUserContext(wallet: ethers.Wallet): Promise<SignerCont
 	return makeSignerContext(userPrimaryWallet, delegateWallet, { days: 1 })
 }
 
-export function makeJpegBlob(fillSize: number): {
+export function makeImageBlob(mimeType: ImageMimeType, fillSize: number): {
```

Comment:
I don't see why you needed to make the mimeType settable in this way to test caching. You could have just sent a different jpeg. Right?